docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -0,0 +1,196 @@
+---
+redirect_from:
+- "/docs/user-guide/persistent-volumes/walkthrough/"
+- "/docs/user-guide/persistent-volumes/walkthrough.html"
+title: Configuring a Pod to Use a PersistentVolume for Storage
+---
+
+{% capture overview %}
+
+This page shows how to configure a Pod to use a PersistentVolume for stoarage.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+You need to have a Kubernetes cluster that has only one Node, and the kubectl
+command-line tool must be configured to communicate with your cluster. If you
+do not already have a single-node cluster, you can create one by using
+[Minikube](/docs/getting-started-guides/minikube).
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Creating an index.html file on your Node
+
+Open a shell to the Node in your cluster. How you open a shell depends on how
+you set up your cluster. For example, if you are using Minikube, you can open a
+shell to your Node by entering `minikube ssh`.
+
+In your shell, create a `/tmp/data` directory:
+
+    mkdir /tmp/data
+
+In the `/tmp/data` directory, create an `index.html` file:
+
+    echo 'Hello from Kubernetes storage' > /tmp/data/index.html
+
+## Creating a PersistentVolume
+
+In this exercise, you create a *hostPath* PersistentVolume. Kubernetes supports
+hostPath for development and testing on a single-node cluster. A hostPath
+PersistentVolume uses a file or directory on the Node that is running the Pod.
+
+In a production cluster, you would not use hostPath. Instead a cluster administrator
+would provision a network resource like a Google Compute Engine persistent disk,
+an NFS share, or an Amazon Elastic Block Store volume.
+
+Here is the configuration file for the hostPath PersistentVolume:
+
+{% include code.html language="yaml" file="task-pv-volume.yaml" ghlink="/docs/tasks/configure-pod-container/task-pv-volume.yaml" %}
+
+The configuration file specifies that the volume is at `/tmp/data` on the
+the cluster's Node. The configuration also specifies a size of 10 gibibytes and
+an access mode of `ReadWriteOnce`, which means the volume can be mounted as
+read-write by a single Node.
+
+Create the PersistentVolume:
+
+    kubectl create -f http://k8s.io/docs/tasks/configure-pod-container/task-pv-volume.yaml
+
+View information about the PersistentVolume:
+
+    kubectl get pv task-pv-volume
+
+The output shows that the PersistentVolume has a `STATUS` of `Available`. This
```

Comment:
Sounds good.  Rethinking this, I suppose a Pod does claim the Volume using a PVC, but the STATUS is independent of whether a Pod is using the claim or not.